.github/workflows/test.yml

```diff
@@ -156,3 +156,25 @@ jobs:
           meta_releases_dict = json.loads('${{ steps.test-set-meta-releases.outputs.meta-releases-dict }}')
           assert isinstance(meta_releases_dict, dict)
           assert len(meta_releases_dict) > 0
+
+  test-install-cylc-components:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v4
+
+      - name: setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
```

Comment:
Sorry, to clarify I meant testing with `setup-micromamba`, although testing with both in a matrix might be good.

Reply:
Have tried, will see if it passes....